adapters/acuityads/acuityadstest/exemplary/banner-app.json

```diff
@@ -0,0 +1,151 @@
+{
+    "mockBidRequest": {
+      "id": "some-request-id",
+      "device": {
+        "ua": "test-user-agent",
+        "ip": "123.123.123.123",
+        "language": "en",
+        "dnt": 0
+      },
+      "tmax": 1000,
+      "user": {
+        "buyeruid": "awesome-user"
+      },
+      "app": {
+        "publisher": {
+          "id": "123456789"
+        },
+        "cat": [
+          "IAB22-1"
+        ],
+        "bundle": "com.app.awesome",
+        "name": "Awesome App",
+        "domain": "awesomeapp.com",
+        "id": "123456789"
+      },
+      "imp": [
+        {
+          "id": "some-impression-id",
+          "tagid": "ogTAGID",
+          "banner": {
+            "w":320,
+            "h":50
+          },
+          "ext": {
+            "bidder": {
+              "host": "ep1",
+              "sourceid": "partner",
+              "accountid": "hash"
```

Comment:
Is `sourceId` an `ext` field that may be of use to your adapter? It doesn't hurt to leave it there since it'll be ignored when unmarshaling `acuityAdsExt` inside `getImpressionExt(imp *openrtb.Imp)`. Just double-checking if this is not a field we want to add to the parameter list.